runtime/j9vm/javanextvmi.cpp

```diff
@@ -302,11 +319,15 @@ JVM_VirtualThreadMountBegin(JNIEnv *env, jobject thread, jboolean firstMount)
 
 	vmFuncs->internalExitVMToJNI(currentThread);
 
+#if JAVA_SPEC_VERSION >= 20
+	virtualThreadHideFrames(currentThread, threadObj, TRUE);
+#endif /* JAVA_SPEC_VERSION >= 20 */
+
```

Comment:
these can be directly added inside the `JVM_VirtualThreadMount/Unmount` functions to avoid duplicates.